opm/output/eclipse/Summary.cpp

```diff
@@ -214,6 +214,7 @@ namespace {
             { "WWIGR", Opm::EclIO::SummaryNode::Type::Rate     },
             { "WMCTL", Opm::EclIO::SummaryNode::Type::Mode     },
             { "WGLIR", Opm::EclIO::SummaryNode::Type::Rate     },
+            { "WGMIR", Opm::EclIO::SummaryNode::Type::Rate     },
```

Comment:
This particular entry is probably unneeded.  The `requiredRestartVectors()` are those that must always be calculated, regardless of presence or absence in the input deck's `SUMMARY` section, because they're needed for simulation restart.  I don't think `WGMIR` is one of those.

I think I'd probably prefer that this entry be removed.

Reply:
Thanks for noticing. I have removed it from the list.